### DISCUSSION TITLE
PROBLEM IN UPLOADING A FILE

### DISCUSSION THREAD

**Discussion by luddifiore** [IBEXImagingCommunity/ibex_imaging_knowledge_base#150] [General]
Hello,
I'm new on IBEX and I encountered a problem while trying to upload a DICOM file. The file contains a sequence of 20 slices (and other softwares, like ITKsnap and 3D Slicer, confirm it), but as soon as I import it on IBEX  it splits into two sequences: one composed by 2 slices and the other by 18. 
Does anyone have some kind of advice to give me to solve the problem?
Thank you :)

**Comment by zivy** [comment]
Hello @luddifiore,

What exactly are you trying to do? If the intent is to contribute information to the knowledge-base, please follow the [how to contribute instructions](https://ibeximagingcommunity.github.io/ibex_imaging_knowledge_base/contrib.html). Note that images in DICOM format will not be uploaded/included in the knowledge-base.
